benchexec/tools/veriabs.py

```diff
@@ -45,9 +45,9 @@ def program_files(self, executable):
     def name(self):
         return "VeriAbs"
 
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        if propertyfile:
-            options = options + ["--property-file", propertyfile]
+    def cmdline(self, executable, options, tasks, rlimits):
+        if tasks.property_file:
+            options += ["--property-file", task.property_file]
         return [executable] + options + tasks
```

Comment:
The new value of `task` is an object that cannot be appended to a list. You probably need to use `[task.single_input_file]`, `[task.input_files]`, or something else depending on what use case your tool supports (cf. the documentation of the `benchexec.tools.template.BaseTool2.Task` class).

Reply:
Yes, Thanks.